### PR TITLE
folder_structure

### PR DESCRIPTION
**Pull request by shinyr**
Hi, Please find the pull request for the branch folder_structure which has the following changes 
1. folder structure 2. scss and converted css file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,8 +2,14 @@
   <head>
     <meta charset="utf-8">
     <title>Responsive Site</title>
+    <link rel="stylesheet" href="assets/css/style.css">
   </head>
   <body>
-
+    <div class ="main">
+      <div class="header">
```
Comment (qburst-neethu) [inline index.html]: Use semantic tag header rather than div tag.

File: index.html [hunk 2]
```diff
@@ -2,8 +2,14 @@
...
+      </div>
+      <div class="footer">
```
Comment (qburst-neethu) [inline index.html]: Use semantic tag footer rather than div tag.

File: index.html (hunk 1 above)
Comment (shinyr) [inline index.html]: @neethumjoy  corrected

File: assets/sass/_reset.scss [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
```
Comment (qburst-neethu) [inline assets/sass/_reset.scss:37]: Avoid the unwanted content property value.

**Pull request merged**